streams/src/main/java/org/apache/kafka/streams/processor/internals/RecordCollectorImpl.java

```diff
@@ -218,22 +228,7 @@ public void onCompletion(final RecordMetadata metadata,
                     }
                 }
             });
-        } catch (final TimeoutException e) {
-            log.error(
-                "Timeout exception caught when sending record to topic {}. " +
-                    "This might happen if the producer cannot send data to the Kafka cluster and thus, " +
-                    "its internal buffer fills up. " +
-                    "This can also happen if the broker is slow to respond, if the network connection to " +
-                    "the broker was interrupted, or if similar circumstances arise. " +
-                    "You can increase producer parameter `max.block.ms` to increase this timeout.",
-                topic,
-                e
-            );
-            throw new StreamsException(
-                String.format("%sFailed to send record to topic %s due to timeout.", logPrefix, topic),
-                e
-            );
-        } catch (final Exception uncaughtException) {
+        }  catch (final Exception uncaughtException) {
```

Comment:
nit: avoid double blank `}  catch` should we single blank `} catch`

Reply:
I am surprised the linter did not catch it before you.